src/Configuration/SshParameters.php

```diff
@@ -126,6 +133,15 @@ public function toArray(): array
 
     public static function fromRaw(array $sshParameters): SshParameters
     {
-        return new SshParameters($sshParameters);
+        return new SshParameters(
+            $sshParameters['enabled'] ?? null,
```

Comment:
jenom prochazim okolo, nemelo by to mit default `false`?  jestli je to bool (coz nazev napovida), tak takhle je tristavovej. Pokud je skutecne tristavovej, tak bych navrhoval [FileNotFound](https://thedailywtf.com/articles/What_Is_Truth_0x3f_) :D